spec/builtin/net/cert_grab_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/net/cert_grab'
+
+describe Ronin::Recon::Net::CertGrab do
+  describe "#process" do
+    context "when there are certificates in the open port" do
+      let(:port)          { Ronin::Recon::Values::OpenPort.new("93.184.216.34", 443, service: 'http', ssl: true) }
+      let(:fixtures_dir)  { File.expand_path(File.join(__dir__,'..','..','values','fixtures')) }
+      let(:cert_path)     { File.join(fixtures_dir,'example.crt') }
+      let(:cert)          { Ronin::Support::Crypto::Cert.load_file(cert_path) }
+
+      it "must yield Cert" do
+        yielded_values = []
+
+        Async do
+          subject.process(port) do |value|
+            yielded_values << value
+          end
+        end
+
+        expect(yielded_values.size).to eq(1)
+        expect(yielded_values[0].cert).to eq(cert)
```

Comment:
Probably still need to add a `expect(yielded_value[0]).to be_kind_of(Ronin::Recon::Values::Cert)`.